frontend/src/metabase/search/components/filters/CreatedByFilter/UserListElement.unit.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import { createMockUserListResult } from "metabase-types/api/mocks";
+import { UserListElement } from "metabase/search/components/filters/CreatedByFilter/UserListElement";
+
+const TEST_USER_LIST_RESULT = createMockUserListResult({
+  common_name: "Alice Johnson",
+});
+
+const TEST_NO_NAME_USER_LIST_RESULT = createMockUserListResult({
+  first_name: "Dave",
+  last_name: "Smith",
+  common_name: undefined,
+});
+
+const setup = ({ value = TEST_USER_LIST_RESULT, isSelected = false }) => {
+  const onClickMock = jest.fn();
+  renderWithProviders(
+    <UserListElement
+      value={value}
+      isSelected={isSelected}
+      onClick={onClickMock}
+    />,
+  );
+  return { onClickMock };
+};
+
+describe("UserListElement", () => {
+  it("should render the component with user's common name", () => {
+    setup({ isSelected: false });
```

Comment:
No need to pass `isSelected: false` param (this comment applies to other `setup` calls as well).